app/views/idv/review/new.html.erb

```diff
@@ -38,11 +33,6 @@
   <div class="text-right margin-top-neg-4 margin-bottom-4">
```

Comment:
Without the accordion, I'd wonder if we want to tighten up the spacing between the submit button. Typically I expect it to be 5 units (2.5rem) between the submit button, which already exists on the button.

```suggestion
  <div class="text-right margin-top-neg-4">
```